keepsorted/keep_sorted_test.go

```diff
@@ -451,8 +463,180 @@ cat`,
 				},
 			},
 		},
+		{
+			name: "NestedBlocks",
+
+			in: `
+// keep-sorted-test start
+foo
+bar
+// keep-sorted-test start
```

Comment:
Suggest adding another line or two to the nested code block, as multiple lines improves coverage slightly (and the below deep nesting case covers the single line case). When dealing with lists, I always just like to have tests for empty, one element, and more than one element.

(We actually don't have a test for an empty block, but there's no reason to ever do that, so who cares.)

Reply:
done